frontends/web/src/hooks/qrcodescanner.ts

```diff
@@ -81,12 +85,20 @@ export const useQRCodeScanner = ({
         })
         .catch(error => {
           if (error) {
+            // Preventing error alerting when user on mobile
+            // navigates from WC connect screen to another page
+            // (this error is 'classified' as NotFoundException).
+            if (error.toString().includes('NotFoundException') && isMobile) {
+              return;
+            }
+            // Otherwise, alert as normal.
             alertUser(error.message || error);
           }
+        }).finally(() => {
           onChangeActiveScanQR(false);
```

Comment:
remove the other `onChangeActiveScanQR(false);` in the first "then"?

Reply:
Good idea but unfort doesn't work well because we need to call this before parseQRResult. Not 100% sure but I think since parseQRreuslt also uses alertUser / dialog, this "clashes" with the currently opened QR scanner dialog and causing the "with id 'video' not found" error. I'll put this back to the `catch` block.